cluster-provision/ocp-cnao/4.4/provision.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -x
```

Comment:
To record it: We can install CNAO into basic OCP and prepull all the images. Particular features then can be enabled by the user of the provider if needed.

Reply:
I am keeping two providers, since they will have different size and you don't want the extra time cost if you don't need CNAO.

Reply:
I especially don't want extra time spending on maintenance of another provider. Could you show what is the actual difference in size? It gets cached on one's host, so I don't see it as that big of an issue. Please prove me wrong.

Reply:
Also, if you happen to be running both providers on your laptop, you end up with both saved on your disk.

Reply:
Let's wait for push size is at quay.io, if it does not worth new provider we just retag it.